### PR TITLE
fix(integration): setup

### PR DESCRIPTION
**Pull request by adrien2p**

### CODE REVIEW

**Comment by adrien2p** [comment]
@riqwan @pKorsholm @olivermrbl could you have a look please. I a currently moving some fixed over there instead of my other pr about the medusa app loader

**Review by pKorsholm — APPROVED** [review]
lgtm!

File: packages/cache-redis/src/services/redis-cache.ts [hunk 1]
```diff
@@ -35,6 +35,10 @@ class RedisCacheService implements ICacheService {
     data: Record<string, unknown>,
     ttl: number = this.TTL
   ): Promise<void> {
+    if (ttl === 0) {
```
Comment (olivermrbl) [inline packages/cache-redis/src/services/redis-cache.ts:38]: note to self: breaking change

**Review by carlos-r-l-rodrigues — APPROVED** [review]

File: packages/cache-inmemory/src/services/inmemory-cache.ts [hunk 2]
```diff
@@ -45,6 +45,10 @@ class InMemoryCacheService implements ICacheService {
    * @param ttl - expiration time in seconds
    */
   async set<T>(key: string, data: T, ttl: number = this.TTL): Promise<void> {
+    if (ttl === 0) {
```
Comment (olivermrbl) [inline packages/cache-inmemory/src/services/inmemory-cache.ts:48]: note to self: breaking change

**Review by olivermrbl — APPROVED** [review]

File: integration-tests/environment-helpers/bootstrap-app.js [hunk 3]
```diff
@@ -2,30 +2,60 @@ const path = require("path")
...
+    port: PORT,
+  }
+}
+
+module.exports = {
+  bootstrapApp,
+  startBootstrapApp: async ({
+    cwd,
+    env = {},
+    skipExpressListen = false,
```
Comment (riqwan) [inline integration-tests/environment-helpers/bootstrap-app.js:39]: How do we handle `verbose: true` here?

File: integration-tests/environment-helpers/use-db.js [hunk 4]
```diff
@@ -168,6 +174,7 @@ module.exports = {
       const options = {
         database: {
           clientUrl: DB_URL,
+          connection: pgConnection,
```
Comment (riqwan) [inline integration-tests/environment-helpers/use-db.js:177]: praise: nice catch

**Review by riqwan — CHANGES_REQUESTED** [review]
Added one question in there, which is pretty important if its not being done at the moment. Its especially important for us to use verbose: true when we need to see whats happening deep in the modules.

This also now breaks the `plugins/__tests__/product/admin/*` specs, which wasn't happening before.

File: integration-tests/environment-helpers/bootstrap-app.js (hunk 3 above)
Comment (adrien2p) [reply integration-tests/environment-helpers/bootstrap-app.js:39]: Verbose was used for the separate process to redirect the output to the main process console here it is on the same process, so you already have the output

Comment (carlos-r-l-rodrigues) [reply integration-tests/environment-helpers/bootstrap-app.js:39]: it isn't another process. logs, etc will be shown by default.

Comment (riqwan) [reply integration-tests/environment-helpers/bootstrap-app.js:39]: Thanks! 🍻

**Review by riqwan — APPROVED** [review]
lgtm, I can pick up the pricing tests at a later point.

**Pull request merged**